doc/source/api.rst

```diff
@@ -621,6 +621,7 @@ Serialization / IO / Conversion
    Series.to_msgpack
    Series.to_json
    Series.to_sparse
+   SparseSeries.to_coo
```

Comment:
Maybe add a section for methods specific to sparse? (just an idea, on the other hand it belongs to the `to_..` methods)